src/state.cairo

```diff
@@ -185,3 +191,84 @@ impl TxOutDefault of Default<TxOut> {
         TxOut { value: 0, pk_script: @"", cached: false, }
     }
 }
+
+#[generate_trait]
+pub impl TransactionImpl of TransactionTrait {
```

Comment:
why did you choose to place `TransactionImpl` and `HeaderImpl` in state.cairo? just by curiosity.

Reply:
That's to be addressed in the next PR :)
But generally it's easier to look up the traits implemented for a particular type if they are in the same file